upgrade/upgrade.go

```diff
@@ -316,6 +323,12 @@ func doResourceUpgrade(namespace string, lhClient *lhclientset.Clientset, kubeCl
 			return err
 		}
 	}
+	if semver.Compare(lhVersionBeforeUpgrade, "v1.8.1") < 0 {
+		logrus.Info("Walking through the resource upgrade path v1.8.0 to v1.8.1")
+		if err := v180to181.UpgradeResources(namespace, lhClient, kubeClient, resourceMaps); err != nil {
```

Comment:
Is here `v180to181.UpgradeResourcesStatus`?

Reply:
You are right! Thank you.